nautobot/core/testing/integration.py

```diff
@@ -28,23 +28,39 @@ class ObjectsListMixin:
     """
 
     def select_all_items(self):
```

Comment:
Mind adding some docstrings to clarify the distinction between `select_all_items`/`select_all_items_from_all_pages`, `click_bulk_delete`/`click_bulk_delete_all`, etc.?